package.json

```diff
@@ -9,22 +9,22 @@
   "scripts": {
     "acceptance": "scripts/acceptance.sh",
     "acceptance-record": "scripts/acceptance_record.sh",
-    "assets": "scripts/assets.sh",
-    "build": "yarn webpack",
+    "assets": "NODE_ENV=production yarn build",
+    "build": "yarn clean && yarn webpack",
     "bundle-report": "NODE_ENV=production ANALYZE_BUNDLE=true yarn build",
     "bundle-report:dev": "NODE_ENV=development ANALYZE_BUNDLE=true yarn build",
-    "clean": "rm -rf public/assets",
+    "clean": "rm -rf public/assets && mkdir -p public/assets",
     "mocha": "scripts/mocha.sh",
     "precommit": "lint-staged",
     "prepare": "patch-package",
     "prepush": "yarn type-check",
     "prettier-project": "yarn run prettier-write 'src/**/*.{ts,tsx,js,jsx}'",
     "prettier-write": "yarn run prettier --write",
-    "publish_assets": "scripts/publish_assets.sh",
+    "publish-assets": "yarn assets && node scripts/uploadToS3.js",
```

Comment:
To my understanding of the new process, we want the manifest and assets built into the image, so we need to put `RUN yarn assets` as a step in the `Dockerfile`.  We can then use a `pre-deploy` hook to run `yarn publish-assets` (which runs the command in the image ready to roll out) [as we do in Positron](https://github.com/artsy/positron/blob/master/hokusai/config.yml#L3)